toolchain/check/testdata/class/no_prelude/name_poisoning.carbon

```diff
@@ -0,0 +1,75 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
```

Comment:
Since this is a test that diagnostics are being generated correctly, I don't think this is a test where including the SemIR is a benefit.
```suggestion
//
// EXTRA-ARGS: --no-dump-sem-ir
//
```

Reply:
Thanks! Makes sense. Done.